Doc/library/socket.rst

```diff
@@ -200,6 +200,23 @@ created.  Socket addresses are represented as follows:
 
   .. versionadded:: 3.8
 
+- :const:`IPPROTO_UDPLITE` is a variant on UDP which allows you to specify
```

Comment:
```suggestion
- :const:`IPPROTO_UDPLITE` is a variant of UDP which allows you to specify
```
on -> of maybe?

Reply:
Sorry, I think that was a regionalism on my part. Fixed.